src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -139,6 +139,23 @@ struct ConsentsUpdater
 };
 
 /**
+ * @brief Structure which contains which actions should be done for specified
+ * application policies iterator
+ */
+struct ActionsForApplicationPolicies {
+  ActionsForApplicationPolicies()
+      : app_policies(NULL)
+      , is_notify_system(false)
+      , is_send_permissions_to_app(false)
+      , is_consent_needed(false) {}
+
+  policy_table::ApplicationPolicies::const_iterator app_policies;
```

Comment:
@AKalinich-Luxoft Why iterator? Why not reference to object? Iterator may be invalidated.

Reply:
@LuxoftAKutsan this field was removed after decomposition and some updates. See https://github.com/smartdevicelink/sdl_core/pull/1779/commits/bfdcdd0f21dc83ffb8a607060d3392cad10406a4